llrt_core/src/modules/console.rs

```
@@ -222,6 +223,11 @@ fn log<'js>(ctx: Ctx<'js>, args: Rest<Value<'js>>) -> Result<()> {
     log_std_out(&ctx, args, LogLevel::Info)
 }
 
+fn clear() -> Result<()> {
```

We are we returning a result if we're ignoring the errors here?